testing/integration/agent_long_test_base_system_integ.json

```diff
@@ -531,7 +531,7 @@
             }
           },
           {
-            "enabled": true,
+            "enabled": false,
```

Comment:
Why are these being disabled?

Reply:
@michel-laterman i actually forgot to update the description. my bad 

Please take a look now.